code/game/objects/items/storage/boxes.dm

```diff
@@ -129,6 +129,10 @@
 	else
 		new /obj/item/tank/internals/plasmaman/belt(src)
 
+	if(HAS_TRAIT(SSstation, STATION_TRAIT_PREMIUM_INTERNALS))
+		new /obj/item/flashlight/flare(src)
+		new /obj/item/radio/off(src)
+
```

Comment:
I'm not sure how I feel about the box itself handling the trait. Could we instead have a global POST_EQUIP signal sent, and then the trait can hook into that, and then insert the items into the survival box?

Reply:
that doesn't sound like a good signal.

Reply:
@coiax you still up in the air about this?

Reply:
@LemonInTheDark Coiax and me couldn't think of a non-overengineered way to fix this for now, so he was okay with leaving it in.